### PR TITLE
Fix prometheus 404 links in documentation

### PR DESCRIPTION
**Pull request by aanm**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5347)

### CODE REVIEW

**Comment by aanm** [comment]
test-docs-please

**Review by techcet — COMMENTED** [review]
Are you sure we want to hard-code 1.1.0 in the microscope yaml link? Also, does this need to be backported to 1.0 and 1.1?

> *[Reviewable](https://reviewable.io/reviews/cilium/cilium/5347#-:-LKhJILBBQUyzLcXzdql:b-wa18ju)* status: :shipit: complete! 0 of 2 files reviewed, all discussions resolved

**Comment by techcet** [comment]
@aanm confirmed that "microscope belongs as different release branches" so choosing to hardcode the link.

File: Documentation/configuration/metrics.rst [hunk 1]
```diff
@@ -74,9 +74,9 @@ Events external to Cilium
 
 Cilium as a Kubernetes pod
 ==========================
-The Cilium `Prometheus reference configuration <https://github.com/cilium/cilium/blob/master/examples/kubernetes/addons/prometheus/prometheus.yaml>`_
-configures jobs that automatically collect pod metrics marked with the
-appropriate two labels.
+The Cilium prometheus reference configuration configures jobs that automatically
```
Comment (techcet) [inline Documentation/configuration/metrics.rst]: Capitalize Prometheus

Comment (techcet) [inline Documentation/configuration/metrics.rst]: ... confguration _that_ configures jobs ...

File: Documentation/configuration/metrics.rst [hunk 2]
```diff
@@ -98,8 +98,8 @@ and tag the resulting metrics series. In the Kubernetes case Prometheus will
 contact the Kubernetes API server for these lists and must have permissions to
 do so.
 
-An example `promethues configuration <https://github.com/cilium/cilium/blob/master/examples/kubernetes/addons/prometheus/prometheus.yaml>`_
-can be found alongside the reference Cilium Kubernetes DaemonSet spec.
+An example of a prometheus configuration can be found alongside the reference
```
Comment (techcet) [inline Documentation/configuration/metrics.rst]: Capitalize Prometheus

**Review by techcet — CHANGES_REQUESTED** [review]

File: Documentation/configuration/metrics.rst (hunk 1 above)
Comment (ianvernon) [reply Documentation/configuration/metrics.rst]: @techcet 

"The Cilium prometheus reference configuration that configures jobs that automatically" sounds grammatically incorrect. I think keeping it as is is OK.

File: Documentation/configuration/metrics.rst (hunk 2 above)
Comment (aanm) [reply Documentation/configuration/metrics.rst]: Done

File: Documentation/configuration/metrics.rst (hunk 1 above)
Comment (aanm) [reply Documentation/configuration/metrics.rst]: @techcet Done

Comment (aanm) [reply Documentation/configuration/metrics.rst]: @techcet Done

**Comment by aanm** [comment]
test-docs-please

**Pull request merged**